openmrs-19/src/main/resources/task-channel.json

```diff
@@ -184,6 +184,24 @@
       ]
     },
     {
+      "displayName": "openMRS.updatePatientIdentifiers",
+      "serviceInterface": "org.motechproject.openmrs19.tasks.OpenMRSActionProxyService",
+      "serviceMethod": "updatePatientIdentifiers",
+      "actionParameters": [
+        {
+          "key": "patientUuid",
+          "displayName": "openMRS.patient.uuid",
+          "required": true
+        },
+        {
+          "key": "identifiers",
+          "displayName": "openMRS.patient.identifiers",
+          "type": "MAP",
+          "required": false
```

Comment:
I think this field should be required, there is no sense in using this action without filling identifiers field